syntax = "proto3";

package chatroom.msgpb;

message RemoveGroup {
    int32 user_id = 1;
    int32 group_id = 2;
}